guides/source/getting_started.md

```diff
@@ -1129,8 +1606,8 @@ you attempt to do just that on the new article form
 We've covered the "CR" part of CRUD. Now let's focus on the "U" part, updating
```

Comment:
By replacing the `new` action with a `show` action, the "C" is no longer covered.

Reply:
This PR does not update this part of the guide, that update will come in a future PR.